arches/app/utils/data_management/sparql_providers/aat_provider.py

```diff
@@ -26,7 +26,7 @@
 class AAT_Provider(Abstract_Provider):
 
     def __init__(self, **kwargs):
-        super(AAT_Provider, self).__init__('http://vocab.getty.edu/sparql.json', **kwargs)
+        super(AAT_Provider, self).__init__('//vocab.getty.edu/sparql.json', **kwargs)
```

Comment:
@veuncent I think that in this case we should retain the protocol (`http:`) as this particular request is sent from the server, rather than the client